govcd/sample_govcd_test_config.json

```diff
@@ -29,14 +29,15 @@
 		"externalIp": "10.150.10.10",
 		"internalIp": "192.168.1.10",
     "disk": {
-      "size": 1024,
-      "sizeForUpdate": 10240
+      "size": 1048576,
+      "sizeForUpdate": 1048576
     }
 	},
 	"logging": {
 		"enabled": true,
 		"logFileName": "go-vcloud-director.log",
 		"logHttpRequests": true,
+		"skipResponseTags": "SupportedVersions,VAppTemplate",
```

Comment:
Didn't we miss here includeFunctionList ?

Reply:
Added